CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Prohibit creating & updating past overrides ([1474](https://github.com/grafana/oncall/pull/1474))
 - Add unique index on `user_id` column in `mobile_app_mobileappauthtoken` table to avoid allowing a user
   to have more than one mobile app auth token at a time ([1482](https://github.com/grafana/oncall/pull/1482))
+- Fixed email sending failure due to newline in title ([81244](https://grafana.com/orgs/allianz1/tickets/81244))
```

Comment:
This should be under the "Unreleased" section of the `CHANGELOG.md`, could you please move it there?

Reply:
Thank you for updating this @vsmanu, may I ask you to also remove `([81244](https://grafana.com/orgs/allianz1/tickets/81244))` part from the changelog?

Reply:
Done